### DISCUSSION TITLE
suggestions for how to handle jupyter notebooks

### DISCUSSION THREAD

**Discussion by pegenaran** [community/community#31831] [New to GitHub]
I have a personal project and I back it up with github. I have multiple jupyter notebooks in it, but if I track them with git, and I run the command git diff, I get a ton formatting changes that are internal to the notebooks, and it makes it difficult to see what changes I made separately in my other files (that are all python code files). I still want the code I set up in the jupyter notebooks tracked for backup, but just not all the extra internal ipython code for the notebooks. Any suggestions? Is there a way to track the jupyter notebooks in a separate git project even though they are in the same directory as my other project files? I am kind of new to github.

**Comment by Rexpert** [comment]
For tracing differences in notebooks, you may give [https://www.reviewnb.com/](https://www.reviewnb.com/) a try. 

![](https://blog.reviewnb.com/assets/images/p6/rnb-diff-comment.png)
